src/can.rs

```diff
@@ -312,6 +360,15 @@ impl Can {
             _tx: tx,
         }
     }
+    /// Initialize the CAN Peripheral
```

Comment:
Minor nit: In this comment you could mention, that the options are implicitly created by `CanOpts::default()`, as I find this information particularly useful to know here. 

Maybe something like  

```suggestion
    /// Initialize the CAN Peripheral
    /// 
    /// The CAN peripheral is configured according to `CanOpts::default()`.
    /// If you want to explicitly set the configuration, use `Can::new_with_opts()` instead.
```